### DISCUSSION TITLE
Can I use the Jetpack preset on PowerShell?

### DISCUSSION THREAD

**Discussion by sshaheen** [starship/starship#6307] [Q&A]
Hi,

I am trying to use the Jetpack preset on PowerShell and I am not seeing the "right-prompt."

![image](https://github.com/user-attachments/assets/a2ab280b-1faf-46a1-9b21-b187483ccbb4)

I was expecting to see information in my prompt like in the example on the [example](https://github.com/starship/starship/blob/master/docs/public/presets/img/jetpack.png).

**Comment by davidkna** [comment]
Unlike many other popular shells, PowerShell does not provide native support for right prompts. This means that starship needs to render them itself. Support for this has been implemented in #4160 and #4208, but has not yet been merged.
